package.json

```diff
@@ -49,6 +49,9 @@
         "node-sass": "^4.11.0",
         "react-facebook-login": "^4.1.1",
         "react-google-login": "^5.0.4",
+        "react-router-hash-link": "^1.2.1",
```

Comment:
Hey @BagzieGracious, what does this library do?

Reply:
@CryceTruly, I have successfully removed this library since I don't use it